### PR TITLE
PL release notes for 787xx

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/early-access.md [hunk 1]
```diff
@@ -12,6 +12,26 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
 
+## March 14, 2023, version 787xx
+
+### Harness Platform
+
+- Harness now populates `givenName` and `familyName` for users via SCIM and returns the same when a GET, CREATE, or UPDATE request is made. (PL-31498)
+
+  This is behind the feature flag `PL_NEW_SCIM_STANDARDS`.
+
+- The response of a CRUD operation on a user or user group now contains the meta fields as per the SCIM 2.0 standards:
```
Comment (KapilDigani) [inline release-notes/early-access.md:23]: contains the following meta fields ?

File: release-notes/harness-platform.md [hunk 2]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 14, 2023, version 787xx
+
+### What's new
+
+- The Harness UI now supports editing the email domain when creating a Service Account. Previously, the email domain was auto-generated and there was no option to edit it. (PL-31769)
+  
+- You can now restrict migration of FirstGen admin users to NextGen by enabling the feature flag `PL_DO_NOT_MIGRATE_NON_ADMIN_CG_USERS_TO_NG`. Previously, all FirstGen admins were migrated as admins and all FirstGen users were migrated with view permissions to NextGen. (PL-31648)
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:26]: Meaning is coming out as wrong .. what we want to say is - now you can control whether you want to migrate all users or just the admin users

File: release-notes/harness-platform.md [hunk 3]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+- The [List Role Assignments by scope filter](https://apidocs.harness.io/tag/Role-Assignments/#operation/getFilteredRoleAssignmentByScopeList) API now supports the following filters:
+
+  - Principal Type Filter: Filters role assignments based on principal type.
+
+  - Harness Managed Filter: Filters role assignments managed by Harness.
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:32]: filters role assignments based on roles managed by harness e.g. Account Admin

File: release-notes/harness-platform.md [hunk 4]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+### Fixed issues
+
+- The template service APIs do not have trace filters. (PL-31829)
+
+  Template service now includes an open telemetry trace filter and the responses have `X-Harness-Trace-ID` in the header.
+
+- The sorting of updated projects fails since the **CreatedAt** field is null when updates are saved. (PL-31794, ZD-40783)
+  
+  A code enhancement has fixed this issue. You can fix this issue in older projects by dummy updating them.
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:67]: by making a dummy update e.g. add a comment

**Review by SudarshanPurohit — APPROVED** [review]
Looks good to me.

File: release-notes/early-access.md (hunk 1 above)
Comment (rashminandasahoo) [reply release-notes/early-access.md]: Done.

File: release-notes/harness-platform.md (hunk 3 above)
Comment (rashminandasahoo) [reply release-notes/harness-platform.md]: Done

File: release-notes/harness-platform.md [hunk 5]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+- Secrets and connectors have different YAML views. (PL-27721)
+
+  UI enhancements have fixed this issue.
+
+- A chained pipeline fails to run because the user cannot enter the codebase branch. (PIE-8720, ZD-40821)
+  
+  A code enhancement has fixed this issue.
+
+- A pipeline becomes unresponsive when invalid YAML is pasted in the run pipeline form. (PIE-8668)
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:94]: Is run pipeline a button? Can we highlight it?

File: release-notes/harness-platform.md [hunk 6]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  UI enhancements have fixed this issue.
+
+- A chained pipeline fails to run because the user cannot enter the codebase branch. (PIE-8720, ZD-40821)
+  
+  A code enhancement has fixed this issue.
+
+- A pipeline becomes unresponsive when invalid YAML is pasted in the run pipeline form. (PIE-8668)
+  
+  The issue is fixed by adding a check for invalid pipeline YAML when pipeline is added via the YAML pipeline studio. 
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:96]: Can we say "The issue has been fixed .."?

File: release-notes/harness-platform.md [hunk 7]
```diff
@@ -17,6 +17,118 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+- Pre-flight check does not work with selective stage execution and pipeline YAML validation fails. (PIE-8476)
+
+  Users can now skip or select pre-flight checks and the pipeline runs successfully.
+
+- After pipeline failure, the console view does not show error details. (PIE-8229)
+  
+  A code enhancement has fixed this issue.
+
+- The backend API to retrieve filtered pipeline executions does not return executions that are marked successful in the UI, but not in the backend. (PIE-8042)
+
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:110]: I didn't understand this. Can we rephrase this?

**Review by SmritiSatya — APPROVED** [review]

File: release-notes/harness-platform.md (hunk 5 above)
Comment (rashminandasahoo) [reply release-notes/harness-platform.md:94]: No, it is not a button.

File: release-notes/harness-platform.md (hunk 6 above)
Comment (rashminandasahoo) [reply release-notes/harness-platform.md:96]: Done

File: release-notes/harness-platform.md (hunk 7 above)
Comment (rashminandasahoo) [reply release-notes/harness-platform.md:110]: Done

**Review by SmritiSatya — DISMISSED** [review]

**Review by roshnisarangadharan — DISMISSED** [review]

**Review by deepakbolangady — APPROVED** [review]
Approving after resolving a merge conflict.

**Pull request merged**